components/financial_assistance/app/views/financial_assistance/applications/verifications/_admin_verification_actions.html.erb

```diff
@@ -1,6 +1,6 @@
 <!-- ***** Verification reason ***** -->
 <div class="row">
-  <div class="verification-update-reason col-md-12" id=<%="#{applicant.id}-#{evidence_type.downcase.split.join('-')}"%>>
+  <div class="verification-update-reason col-md-12" id=<%="#{applicant.id}-#{evidence_type&.downcase&.split&.join('-')}"%>>
```

Comment:
I realize you're just fixing this, but this looks like potentially too much logic in the view, can we move this to a method that returns the appropriate value and just use the method here instead?